Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,249 @@
+
+   .. include:: /Documentation/beta.rst
+
+Upgrade notes
+~~~~~~~~~~~~~
+
+When the PR introduces new user-facing options, metrics, or behavior that
+affects upgrades or downgrades, ensure that the author summarizes the changes
+with a note in ``Documentation/operations/upgrade.rst``.
+
+Completeness
+~~~~~~~~~~~~
+
+Make sure that new or updated content is complete, with no TODOs.
+
+Auto-generated reference documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When certain parts of the Cilium repository change, contributors may have to
+update some auto-generated reference documents that are part of Cilium's
+documentation, such as the command reference or the Helm reference. The CI
+validates that these updates are present in the PR. If they are missing, you
+may have to help contributors figure out what commands they need to run to
+perform the updates. These commands are usually provided in the logs of the
+GitHub workflows that failed to pass.
+
+Spell checker exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Documentation checks include running a spell checker. This spell checker
+uses a file, ``Documentation/spelling_wordlist.txt``, containing a list of
+spelling exceptions to ignore. Team cilium/docs-structure is the owner for this
+file. Usually, there is not much feedback to provide on updates to the list of
+exceptions. However, it's useful for reviewers to know that:
+
+  - Entries are sorted alphabetically, with all words starting with uppercase
+    letters coming before words starting with lowercase letters.
+  - Entries in the list of exceptions must be spelled correctly.
+  - Lowercase entries are case-insensitive for the spell checker, so reviewers
+    should reject new entries with capital letters if the lowercase versions
+    are already in the list.
+
+Netlify preview
+~~~~~~~~~~~~~~~
+
+`Netlify`_ builds a new preview for each PR touching the documentation. You are
+not expected to check the preview for each PR. However, if the PR contains
+detailed formatting changes, such as nested blocks or directives, or changes to
+tables or tabs, then it's good to validate that changes render as expected.
+Also check the preview if you have a doubt as to the validity of the
+reStructuredText (RST) mark-up that the author uses.
+
+The list of checks on the PR page contains a link to the Netlify preview. If
+the preview build failed, the link leads to the build logs.
+
+.. _Netlify: https://www.netlify.com/?attr=homepage-modal
+
+Formatting
+----------
+
+Read :ref:`Cilium's documentation style guide <docs_style_guide>`.
+
+Flag poor formatting or obvious mistakes. The syntax for RST is not always
+trivial and some contributors make mistakes, or they simply forget to use RST
+and they employ Markdown mark-up instead. Make sure authors fix such issues.
+
+Keep an eye on :ref:`code-blocks <docs_style_code_blocks>`: do they include RST
+substitutions, and if so, do they use the right directive? If not, do they use
+the right language?
+
```

Comment:
Not sure whether it's specifically worth mentioning here or if we think it's covered by "read the style guide" but callouts are another low-hanging fruit that is worth keeping vigilant over. I think that this is pretty well understood across @cilium/docs-structure already today, but if we think it's helpful to also explicitly mention here then by all means let's do so. I don't have a strong opinion about this.

Reply:
Callouts are also explicitly mentioned on this page, but earlier, under "Documentation structure" (given that they're not just formatting, but more a matter of how the new info is incorporated into the document).